src/renderers/shaders/ShaderChunk/fog_fragment.glsl

```diff
@@ -1,5 +1,7 @@
 #ifdef USE_FOG
 
+	float fogDepth = length(fogPosition);
```

Comment:
nomenclature and formatting
```glsl
float fogDepth = length( vFogPosition );
```

Reply:
I'll merge and clean up.

Reply:
The other variables on the file do not use that nomenclature. Will keep it as `fogPosition` for now.

Reply:
I know... The implication was to change all occurrences to match the nomenclature convention.